src/modules/standard-clauses/standard-clauses.service.ts

```
@@ -3,6 +3,8 @@ import { InjectRepository } from '@nestjs/typeorm';
 import { Repository } from 'typeorm';
 import { StandardClause } from '../../entities/standard-clause.entity';
 import { CreateStandardClauseDto } from './dto/create-standard-clause.dto';
+import { UpdateStandardClauseDto } from './dto/update-standard-clause.dto';
+import { UpdateStandardClauseDto } from './dto/update-standard-clause.dto';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'UpdateStandardClauseDto'. Consider to delete it or rename it.

'UpdateStandardClauseDto' is defined here:


(lint/suspicious/noRedeclare)